docs/reference/method/MongoDBCollection-createSearchIndex.txt

```diff
@@ -0,0 +1,91 @@
+========================================
+MongoDB\\Collection::createSearchIndex()
+========================================
+
+.. versionadded:: 1.17
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Definition
+----------
+
+.. phpmethod:: MongoDB\\Collection::createSearchIndex()
+
+   Create an Atlas Search index for the collection.
+
+   .. code-block:: php
+
+      function createSearchIndex(array|object $definition, array $options = []): string
+
+   This method has the following parameters:
+
+   .. include:: /includes/apiargs/MongoDBCollection-method-createSearchIndex-param.rst
+
+   The ``$options`` parameter supports the following options:
+
+   .. include:: /includes/apiargs/MongoDBCollection-method-createSearchIndex-option.rst
+
+   .. note::
+
+      This command can only be run on a deployment hosted on
+      :manual:`MongoDB Atlas </atlas>` and requires an Atlas cluster tier of at
+      least M10.
```

Comment:
It's now also available locally, I don't know if we want to mention that: https://www.mongodb.com/blog/post/introducing-local-development-experience-atlas-search-vector-search-atlas-cli

Reply:
Is there an actual documentation page for local development? I don't want to link to the blog from our documentation, as I consider that content transient and not a canonical source of truth.

Reply:
Here it is: https://www.mongodb.com/docs/atlas/cli/stable/atlas-cli-deploy-local/#std-label-atlas-cli-deploy-fts

Reply:
I added a note on the create/update methods.